### PR TITLE
Basic footer

### PR DESCRIPTION
**Pull request by khushichitalia**
links to pages in progress

### CODE REVIEW

**Comment by ethanniser** [comment]
the error was that you put the footer in `./client/index.tsx` which is a bit too high up in the tree

the layout for the page (where the footer goes) is in `./client/routes/__root.tsx`

but it looks good just needs tailwind (chatgpt should be able to help here)

File: client/components/footer.tsx
Comment (ufsasewebmaster) [inline client/components/footer.tsx:1]: We want to use tailwind CSS (?), and you can reduce a lot of repetition here. Make sure to keep in mind code readability.

File: client/components/ui/footer.tsx
Comment (ufsasewebmaster) [inline client/components/ui/footer.tsx:1]: Make sure you review your errors/warning before pushing

**Review by ufsasewebmaster — APPROVED** [review]

**Comment by ufsasewebmaster** [comment]
Could you update this after pulling from sprint-0?

**Pull request merged**